package.json

```diff
@@ -57,6 +57,7 @@
     "css-prefix-main": "postcss --config build/postcss.config.js --replace \"dist/css/*.css\" \"!dist/css/*.rtl*.css\" \"!dist/css/*.min.css\"",
     "css-prefix-examples": "postcss --config build/postcss.config.js --replace \"site/content/**/*.css\"",
     "css-prefix-examples-rtl": "cross-env-shell NODE_ENV=RTL postcss --config build/postcss.config.js --dir \"site/content/docs/$npm_package_config_version_short/examples/\" --ext \".rtl.css\" --base \"site/content/docs/$npm_package_config_version_short/examples/\" \"site/content/docs/$npm_package_config_version_short/examples/{blog,carousel,dashboard,cheatsheet}/*.css\" \"!site/content/docs/$npm_package_config_version_short/examples/{blog,carousel,dashboard,cheatsheet}/*.rtl.css\"",
+    "css-test": "mocha --config scss/tests/.mocharc.js",
```

Comment:
Please see if we can use jasmine. A new dependency as mocha where they pin all their deps is pretty annoying.

Reply:
Sure, I'll rework that to use Jasmine. Hopefully there'll be some overlap between its dependencies and the one pulled by Karma 😊 I could try set it to the same version pulled by Karma, if looking for as much overlap as possible. But it feels like it's something that'll be easily missed on a future upgrade...